Resources/Prototypes/_RMC14/Entities/Objects/Weapons/Guns/Ammunition/Projectiles/shotgun.yml

```diff
@@ -95,3 +95,20 @@
   name: incendiary buckshot shell
   #components: #TODO RMC14 add burning
 
+- type: entity
+  parent: CMPelletShotgunBase
+  id: RMCPelletShotgunBreaching
+  name: breaching shell
+  components:
+  - type: Sprite
+    sprite: _RMC14/Objects/Weapons/Guns/Ammunition/Projectiles/shotgun_projectiles.rsi
+    layers:
+    - state: flechette
+      shader: unshaded
+  - type: Projectile
+    damage:
+      types:
+        Blunt: 25
+        Structural: 500
```

Comment:
Why does this do structural? And blunt? Our bullets normally do piercing. Projectile stats from CM:

![projectile](https://github.com/user-attachments/assets/5406ad55-0286-4cbc-bf8c-5064d35815a3)

We'll be able to use `RMCExtraProjectilesDamageModsComponent` to adjust the additional projectiles' damage once #3422 is merged.

Reply:
I'm relying on the damage numbers from cm13 and making them approximate, and also as far as I know I can't run 2 different projectiles at once at the same time
![image](https://github.com/user-attachments/assets/d87e29b3-0b4e-46f2-801a-19c77e7d75de)